polaris/src/test/java/com/synopsys/integration/polaris/common/service/FullTest.java

```diff
@@ -19,38 +22,40 @@
 import com.synopsys.integration.polaris.common.model.IssueResourcesSingle;
 
 public class FullTest {
-    public static void main(final String[] args) throws IntegrationException {
-        final PolarisServerConfigBuilder polarisServerConfigBuilder = PolarisServerConfig.newBuilder();
+    @Test
+    @Ignore
```

Comment:
If we ignore the test, should we just remove it?

Reply:
I only kept it as an ignored test so that a user could always run the test manually in the IDE if needed.  When Polaris support is removed from detect then this class can be removed.